packages/js/src/components/link-suggestions.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from "prop-types";
+
+import styled from "styled-components";
+import { __, sprintf } from "@wordpress/i18n";
+import { createInterpolateElement } from "@wordpress/element";
+import { makeOutboundLink } from "@yoast/helpers";
+
+import LinkSuggestion from "./link-suggestion";
+
+const introMessage =  createInterpolateElement(
+	sprintf(
+		/**
+		 * translators: %1$s expands to an opening anchor tag.
+		 * %2$s expands to a closing anchor tag.
+		 */
+		__( "To improve your site structure, consider linking to other relevant posts or pages on your website. %1$sRead our guide on internal linking for SEO%2$s to learn more.", "wordpress-seo" ),
+		"<a>",
+		"</a>"
+	),
+	{
+		// eslint-disable-next-line jsx-a11y/anchor-has-content, jsx-a11y/anchor-is-valid
+		a: <a href="#" target="_blank" rel="noreferrer" style={ { cursor: "default", pointerEvents: "none" } } />,
+	}
+);
+
+const OutboundLink = makeOutboundLink();
+const upsellLink = "http://yoa.st/link-suggestions-upsell";
```

Comment:
> In the 'fake' links we do show the 'Upgrade to' for example as a link but not clickable. Should we make this the same as the upsell link?

No, we just want to show it as it would be with Premium: making that link clickable would be confusing 🙂 

>I am missing the 'premium' badge from the design in the issue for the sidebar. Is this something we need to add or was it decided to skip this?

In the issue's description Kai added: _Please note: in the designs there’s a Premium badge in the collapsible heading. It was decided to not add that badge, so please leave that out._. So, no badge for us 😁 

>I am wondering if we need another link for elementor, because now the url has &utm_content=editor in the deeplink and not sure if that is OK

Problem is the shortlinker implementation JS-side is far from optimal, and giving the fact we get the editor information from other links, it's ok to leave it as it is now.

>Should we also add the extra data like your php version and such here. And I also see that the link is http instead of https

Same as above, only thing is I'll add the `s` to the `http` just to uniform it with the code: in reality the shortlink redirects to an https address anyway.